README.md

```diff
@@ -5,40 +5,49 @@
-| [MultiTenancy Schema Based](./jpa/multitenancy/schema)                                                                  | The application, demonstrates running multi tenancy in JPA using schema based i.e Shared Database with Separate Schema                                                                                                                                              | Completed |
-| [MultiTenancy with multipledatsources](./jpa/multitenancy/multidatasource-multitenancy)                                 | The application, demonstrates running multi tenancy in JPA using all strategies using multidatasources                                                                                                                                                              | Completed |
-| [spring-boot-graphql-querydsl](./graphql/boot-graphql-querydsl)                                                         | The application, demonstrates the way to connect to database using graph ql and querydsl                                                                                                                                                                            | WIP       |
-| [spring-boot-graphql-webmvc](./graphql/boot-graphql-webmvc)                                                             | The application, demonstrates how to apply graphql concepts to the spring boot project                                                                                                                                                                              | WIP       |
-| [Custom SequenceNumber and LazyConnectionDataSourceProxy for db connection improvement](./jpa/boot-data-customsequence) | This application demonstrates: Custom sequence generation, Database connection optimization using datasource-proxy and LazyConnectionDataSourceProxy with MariaDB, SQL query validation using SQLStatementCountValidator, Dynamic validation using ValidationGroups | Completed |
-| [KeySet pagination and dynamic search](./jpa/keyset-pagination/blaze-persistence)                                       | Implements KeySet Pagination using Blaze Persistence and enable dynamic search using specifications                                                                                                                                                                 | Completed |
+| Name                                                                                                                    | Description 		                                                                                                                                                                                                                                                                                                                                                              | Status 		 |
```

Comment:
@coderabbitai, please review the Description of all rows in this table and suggest improvements to make is more clear

Reply:
@coderabbiai, can you please help me to update all ReadMe files in the submodule so anyone can understand what that project is doing and provides as much details as possible.